src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private final Map<String, Integer> fruits = new HashMap<>();
+
+    public Map<String, Integer> getFruits() {
+        return fruits;
+    }

```

Comment:
we dont want to create multiple storages during program run
```suggestion
    private static final Map<String, Integer> fruits = new HashMap<>();

    public static Map<String, Integer> getFruits() {
        return fruits;
    }
```